### DISCUSSION TITLE
Change "new quest" label to "new quest, accepted" label?

### DISCUSSION THREAD

**Discussion by matkoniecz** [streetcomplete/StreetComplete#3840] [Repository Maintenance]
[`new quest` label](https://github.com/streetcomplete/StreetComplete/issues?q=is%3Aissue+is%3Aopen+label%3A%22new+quest%22) has a bit hidden meaning "someone with label rights looked at it and approved this quest idea"

Maybe it would be worth making this more explicit?

Or maybe it would just be more problematic where someone adds this purple label and @westnordost has a different idea and removes it?

Raised by @wielandb in https://github.com/streetcomplete/StreetComplete/pull/3789#pullrequestreview-890235838

I plan on editing this label, but I want to ask first to give people chance to respond and avoid multiple mass-changes of labels

**Comment by FloEdelmann** [comment]
Another idea would be a new `accepted` label that could go alongside `new quest` or `enhancement`.

**Reply by westnordost** [reply]
Anything that is marked with anyo f these two labels is "accepted"

**Reply by mnalis** [reply]
True, but that local project policy (that `new quest` and `enhancement` imply `accepted`) is not widely known by most of the GitHub visitors, which has caused confusion in several discussions. 

I guess that provoked this idea to try to improve on the situation in order to reduce visitor confusion. 

- from purity standpoint I prefer @FloEdelmann idea of separate `accepted` tag, but that unfortunately somewhat increases the load on people setting the tags (and is prone to human error)
- from lets-not-increase-westnordost-load point, @matkoniecz idea of naming tags like `new quests, accepted` is uglier, but keeps the load / workflow unchanged (and avoids the human error of forgetting to add one tag like idea above).
- of course, there is always the option of not changing anything about tags (but merely documenting that policy somewhere if it isn't already), but that has not proven robust enough to avoid confusion so far.

**Reply by westnordost** [reply]
Many quests that are kind of accepted but are somewhat problematic, complex to implement or kind of are waiting for research / tagging discussions are also marked as `new quest`. Some, which are completely blocked (by missing OSM consensus) are additionally labeled with `blocked` or closed entirely, but not all.

In my perception, this has more often lead to confusion. Renaming it to "accepted" may even make this more of an issue than it is now.

**Comment by westnordost** [comment]
So what is your idea to rename it?

**Reply by mnalis** [reply]
The title suggests changing label from `new quest` to `new quest, accepted` which sounds reasonable to me (also, by same logic, `enhancement` => `enhancement, accepted` might be done)

**Reply by westnordost** [reply]
Oh, I overread that. Well, I don't really like that. Will we have "bug, acknowledged", etc. then too?
